pkgs/by-name/in/inhibridge/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitea,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "inhibridge";
+  version = "0.3.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "Scrumplex";
+    repo = "inhibridge";
+    rev = version;
+    hash = "sha256-cKVw3Gd4Ml8BeXjZqTN6ToeRzO9PI+Sn45gpltlRuWM=";
+  };
+
+  cargoHash = "sha256-uKSbxAsGUR2nYfdtiTR/bgPBQunqYLzx3+CmszNilPQ=";
+
+  meta = with lib; {
```

Comment:
Remove `with ;`
See: https://github.com/NixOS/nixpkgs/issues/208242

Reply:
As per Sandro's comment in that issue, with lib in small scopes like meta is fine